setup.cfg

```diff
@@ -19,3 +19,5 @@ install_requires =
   genpw
   python-dotenv
   redis
+  # waiting for pip/setuptools to support custom package indexes:
+  # pip install -i https://pypi.drlazor.be metrics
```

Comment:
But there is currently no way to setup setuptools so it fetches one package from an other index... I mean there was a way back in pip v18 but `dependencies_link` are no more supported since v19. I'm sorry guys but you'll need to manually install metrics `pip install -i https://pypi.drlazor.be metrics`.